File: 260127/약수가 세 개인 수/numbers-with-three-divisors.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc=new Scanner(System.in);

        int start = sc.nextInt();
        int end = sc.nextInt();

        int count=0;

        for(int i=start; i<end; i++){
            int div_count=0;
            for(int d=1; d<i+1; d++){
                if(i%d==0) div_count++;
            }
            if(div_count==3){
                count++;
            }
        }
        System.out.println(count);
    }
}

File: 260304/약수가 세 개인 수/numbers-with-three-divisors.cpp
#include <iostream>

using namespace std;

int st, ed;

int count=0;

void divisorCount(int num){
    int check=0;
    for(int i=1; i<num+1; i++){
        num%i==0?check++:false;
    }
    check==3?count++:false;
}

int main() {
    cin >> st >> ed;

    // Please write your code here.
    for(int i=st; i<ed+1; i++){
        divisorCount(i);
    }

    cout<<count;

    return 0;
}
